### PR TITLE
obc status and descriptions added

### PR DESCRIPTION
**Pull request by vijaykanthm28**

### CODE REVIEW

File: megdc/constants.go [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+	TPL_ONEMASTER_FINISHED   = "onemaster.finished"
+
+	TPL_ONEHOST_RUN       = "onehost.running"
+	ONEHOST_PEPARE_BEGIN  = "onehost.depends_install.starting"
+	ONEHOST_PREPARE_END   = "onehost.depends_install.completed"
+	ONEHOST_INSTALL_BEGIN = "onehost.install.starting"
+	ONEHOST_INSTALL_END   = "onehost.install.completed"
+	TPL_ONEHOST_FINISHED  = "onehost.finished"
+
+	TPL_CEPHCLUSTER_RUN          = "cephculster.running"
```
Comment (rajthilakmca) [inline megdc/constants.go]: correct the "cephculster.running" spell mistakes

File: megdc/constants.go [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+
+	TPL_ONEMASTER_RUN        = "onemaster.running"
+	ONEMASTER_INSTALL_BEGIN  = "onemaster.install.starting"
+	ONEMASTER_INSTALL_END    = "onemaster.install.completed"
+	ONEMASTER_ACTIVATE_BEGIN = "onemaster.activate.starting"
+	ONEMASTER_ACTIVATE_END   = "onemaster.activate.completed"
+	TPL_ONEMASTER_FINISHED   = "onemaster.finished"
+
+	TPL_ONEHOST_RUN       = "onehost.running"
+	ONEHOST_PEPARE_BEGIN  = "onehost.depends_install.starting"
```
Comment (rajthilakmca) [inline megdc/constants.go]: use "." instead of "_" and use meaning full events text

Comment (rajthilakmca) [inline megdc/constants.go]: please re-check all events

File: megdc/status.go [hunk 3]
```diff
@@ -0,0 +1,391 @@
...
+type State string
+
+func (s State) String() string {
+	return string(s)
+}
+
+func (s Status) Event_type() string {
+	switch s.String() {
+	case TPL_HOSTINFOS_RUN:
+		return OBCHOSTINFOSTPLRUN
```
Comment (rajthilakmca) [inline megdc/status.go]: find some generic way to get a string instead of switch case

**Comment by indykish** [comment]
@rajthilakmca  Kindly review and merge

**Pull request merged**